stdlib/public/core/ManagedBuffer.swift

```diff
@@ -101,20 +106,7 @@ open class ManagedBuffer<Header, Element> {
   /// `ManagedBuffer.create`'s call to initialize, `ManagedBuffer`'s
   /// `header` property is as-yet uninitialized, and therefore
   /// reading the `header` property during `ManagedBuffer.create` is undefined.
-  public final var header: Header {
-    addressWithNativeOwner {
-      return (
-        ManagedBufferPointer(self).withUnsafeMutablePointerToHeader {
-          UnsafePointer($0)
-        },
-        Builtin.castToNativeObject(self))
-    }
-    mutableAddressWithNativeOwner {
-      return (
-        ManagedBufferPointer(self).withUnsafeMutablePointerToHeader { $0 },
-        Builtin.castToNativeObject(self))
-    }
-  }
+  public final var header: Header
```

Comment:
With tail-allocated arrays in SIL we can now really _store_ the header in the class. Before we had an "empty" class and did these address calculation tricks to pretend there is a header. Therefore the "header" was a computed property.